picard/album.py

```diff
@@ -469,7 +472,7 @@ def _remove_file(self, track, file):
         file.metadata_images_changed.disconnect(self.update_metadata_images)
         remove_metadata_images(self, [file])
 
-    def _match_files(self, files, recordingid=None, threshold=0):
+    def _match_files(self, files, recordingid=None, threshold=0, debug=True):
```

Comment:
Why the `debug` parameter here? I don't see it used anywhere, and it is just guarding a call to `log.debug` below.

Reply:
Because i want to make this testable, and log.debug() is just for output, perhaps not the correct approach though.

Reply:
Finally i remove the `debug` parameter and associated debug output, they aren't strictly needed.
See https://github.com/metabrainz/picard/pull/1144/commits/3bd05efadac0c8776737b94c9c1be100eacc7607